corehq/apps/app_manager/translations.py

```diff
@@ -446,6 +452,10 @@ def update_form_translations(sheet, rows, missing_cols, app):
     :return:  Returns a list of message tuples. The first item in each tuple is
     a function like django.contrib.messages.error, and the second is a string.
     """
+    def value_is_safe(value_element):
+        is_safe = lambda el: isinstance(el, etree.Entity) or isinstance(el, etree.Element) and el.tag in SAFE_TAGS
```

Comment:
Ah! Thank you. (An entity is like `&#234;`, e.g.

    >>> etree.Entity("#234")
    &#234;

)